config_template.cfg

```diff
@@ -131,21 +169,22 @@ REF_AREA= 1.0
 % Aircraft semi-span (0 implies automatic calculation) (m or in)
 SEMI_SPAN= 0.0
 %
-% Flow non-dimensionalization (DIMENSIONAL, FREESTREAM_PRESS_EQ_ONE,
+% Compressible flow non-dimensionalization (DIMENSIONAL, FREESTREAM_PRESS_EQ_ONE,
 %                              FREESTREAM_VEL_EQ_MACH, FREESTREAM_VEL_EQ_ONE)
 REF_DIMENSIONALIZATION= DIMENSIONAL
```

Comment:
To be consistent with INC_NONDIM should we call this option COMP_NONDIM?

Reply:
We should agree in general on a consistent naming scheme for these options. And we should maybe move that option to the compressible free stream definition section.

Reply:
While the compressible and incompressible both use the same enumerated type (it will throw an error if we redefine DIMENSIONAL, for instance), I have used a separate option, INC_NONDIM, for the incompressible non-dim options, which are only DIMENSIONAL, INITIAL_VALUES (default), or specified REFERENCE_VALUES. I separate the option because we may need to supply the INC_*_REF values, and I wanted all of the incompressible options in the config file to be consistent with the INC_* prefix. 

I also did not change the name of the compressible option, REF_NONDIMENSIONALIZATION, since this would break many config files. I did, however, move this option into the compressible section of the config template.

I am open to more comments on this one.. but these were my thoughts.